app/src/main/java/com/github/se/polyfit/ui/components/button/LikeButton.kt

```diff
@@ -0,0 +1,56 @@
+package com.github.se.polyfit.ui.components.button
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Favorite
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.FilledTonalButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun LikeButton(
+    likesCount: Long,
+    modifier: Modifier = Modifier,
+    onLikeClicked: (Boolean) -> Unit = {}
+) {
+  FilledTonalButton(
+      onClick = { onLikeClicked },
+      colors = ButtonDefaults.buttonColors(containerColor = Color.White),
+      shape = RoundedCornerShape(20.dp),
+      enabled = false, // Set this to true
+      modifier = modifier.padding(horizontal = 25.dp, vertical = 8.dp).testTag("LikeButton")) {
```

Comment:
It's usually suggested in design/ui to follow an 8 point grid (https://uxplanet.org/everything-you-should-know-about-8-point-grid-system-in-ux-design-b69cb945b18d), so its best to use multiples of 8.
```suggestion
      modifier = modifier.padding(horizontal = 24.dp, vertical = 8.dp).testTag("LikeButton")) {
```

Reply:
Oh, I never knew that. Thanks for linking an article; I'll read it right now!